.github/ISSUE_TEMPLATE/article-needs-fixing

```diff
@@ -0,0 +1,23 @@
+---
+name: 'Bug Report: Issues with \news or guide articles or documentation'
+about: Report issue with a specific article, like broken links, typos, missing parts, etc
+title: ''
+labels: 'status: needs help for triage, platform: news, platform: forum'
```

Comment:
```suggestion
labels: 'status: needs help for triage, type: bug'
```

The issue would need triage on which team (platform) should be addressing it. Also bug is the apt label.